icons/premid.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>PreMiD</title><path d="M22.44 4.24H1.56A1.56 1.56 0 0 0 0 5.8v12.4a1.56 1.56 0 0 0 1.56 1.56h20.88A1.56 1.56 0 0 0 24 18.2V5.8a1.56 1.56 0 0 0-1.56-1.56m-9.506 4.851a.72.72 0 0 1 .509-.21h4.64a.72.72 0 0 1 0 1.438h-4.644a.72.72 0 0 1-.509-1.229Zm-3.27 3.244-6.085 3.51A.386.386 0 0 1 3 15.511V8.489a.386.386 0 0 1 .58-.334l6.08 3.51a.386.386 0 0 1 0 .67zm11.885 2.574a.72.72 0 0 1-.509.21h-7.601a.72.72 0 0 1 0-1.438h7.601a.72.72 0 0 1 .509 1.229zm0-2.4a.72.72 0 0 1-.509.211h-7.601a.72.72 0 0 1 0-1.439h7.601a.72.72 0 0 1 .509 1.229z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="265" height="175" viewBox="0 0 265 175"><path fill="#FFF" fill-rule="evenodd" d="M372,169 C381.388841,169 389,176.611159 389,186 L389,327 C389,336.388841 381.388841,344 372,344 L141,344 C131.611159,344 124,336.388841 124,327 L124,186 C124,176.611159 131.611159,169 141,169 L372,169 Z M162,217.67878 C157.581722,217.67878 154,221.260502 154,225.67878 L154,288.32122 C154,289.695115 154.353828,291.045857 155.027396,292.24331 C157.193506,296.094173 162.071227,297.459935 165.92209,295.293824 L221.604259,263.972604 C222.881255,263.254294 223.936463,262.199086 224.654773,260.92209 C226.820884,257.071227 225.455122,252.193506 221.604259,250.027396 L165.92209,218.706176 C164.724637,218.032608 163.373895,217.67878 162,217.67878 Z M351.5,277 L266.5,277 C262.357864,277 259,280.357864 259,284.5 C259,288.642136 262.357864,292 266.5,292 L351.5,292 C355.642136,292 359,288.642136 359,284.5 C359,280.357864 355.642136,277 351.5,277 Z M351.5,250 L266.5,250 C262.357864,250 259,253.357864 259,257.5 C259,261.642136 262.357864,265 266.5,265 L351.5,265 C355.642136,265 359,261.642136 359,257.5 C359,253.357864 355.642136,250 351.5,250 Z M316.5,223 L266.5,223 C262.357864,223 259,226.357864 259,230.5 C259,234.642136 262.357864,238 266.5,238 L316.5,238 C320.642136,238 324,234.642136 324,230.5 C324,226.357864 320.642136,223 316.5,223 Z" transform="translate(-124 -169)"/></svg>
```

Reply:
You might need to run it through svgo again, apparently it didn't quite work for me

Reply:
Done!